public/index.js

```diff
@@ -174,16 +177,26 @@ const clearSuccessAlert = () => {
 
 $('#url, #title, #folder-select').on('keyup', () =>{
   enableCheck()
-  // clearSuccessAlert()
 })
 
 $('#folders-holder').on('click', '.retrieved-folder', function(e) {
   const parsedId = parseInt(this.id, 10)
 
-  getUrlsByFolder(parsedId)
+  const selectedFolder = $( this )
```

Comment:
I have a couple ideas I'll try to figure this out. It's better late than never, but at least it mostly works!